librocksdb_sys/crocksdb/crocksdb/c.h

```diff
@@ -1651,6 +1655,24 @@ crocksdb_compact_files_cf(crocksdb_t*, crocksdb_column_family_handle_t*,
                           int output_level,
                           char** errptr);
 
+extern C_ROCKSDB_LIBRARY_API crocksdb_options_t*
+crocksdb_get_db_options(crocksdb_t* db);
+
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_db_options(crocksdb_t* db,
+                        const char** names,
+                        const char** values,
+                        size_t num_options,
+                        char** errptr);
+
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_cf_options(crocksdb_t* db,
```

Comment:
Should we add `get_cf_options`?

Reply:
We have, but it's called `get_options_cf`.

Reply:
Maybe change it to `get cf options` and put them together if it's ok.

Reply:
We can't change the name for compatibility reason, but I have put them together.